### PR TITLE
Add info on local sphinx build

### PR DESCRIPTION
**Pull request by stijnvanhoey**
* [x] I have checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change
* [x] I made sure all new and existing data sets have a description in the `data/README.md` file
* [x] I have updated the documentation accordingly.

This PR provides a bit more background information on the Sphinx setup and how a local run should work. @Beramos, can you have a check?

### CODE REVIEW

File: docs/collaborate.md [hunk 1]
```diff
@@ -43,11 +43,22 @@ __Note:__ Use one of the `notebook/templates` to get started. You can run these
...
+```
+
+An alternative (for linux users only) is using the `Makefile` provided with Sphinx itself. To build the html-pages:
+
+```
+cd docs
+make html
+```
+
+The resulting html-website is created in the directory `build/sphinx/html`. Double click any of the `html` file in the folder to open the website in your browser (no running server required).
```
Comment (jorisvandenbossche) [inline docs/collaborate.md:61]:
```suggestion
The resulting html-website is created in the directory `docs/_build/html`. Double click any of the `html` file in the folder to open the website in your browser (no running server required).
```

File: docs/collaborate.md [hunk 2]
```diff
@@ -43,11 +43,22 @@ __Note:__ Use one of the `notebook/templates` to get started. You can run these
 
 Documentation consists of both the technical matter about the code as well as background information on the models. To keep these up to date and centralized, we use [Sphinx](https://www.sphinx-doc.org/en/master/) which enables us to keep the documentation together on a website.
 
-The Sphinx setup provides the usage of both `.rst` file, i.e. [restructuredtext](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html) as well as `.md` files, i.e. [Markdown](https://www.markdownguide.org/basic-syntax/). The latter is generally experienced as easier to write, while the former provides a more advanced functionalities.
+The Sphinx setup provides the usage of both `.rst` file, i.e. [restructuredtext](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html) as well as `.md` files, i.e. [Markdown](https://www.markdownguide.org/basic-syntax/). The latter is generally experienced as easier to write, while the former provides a more advanced functionalities. Existing pages can be adusted directly (editing them online or on your computer). When you want to create a new page, make sure to add the page to the `index.rst` in order to make the page part of the website.
```
Comment (Beramos) [inline docs/collaborate.md:46]:
```suggestion
The Sphinx setup provides the usage of both `.rst` file, i.e. [restructuredtext](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html) as well as `.md` files, i.e. [Markdown](https://www.markdownguide.org/basic-syntax/). The latter is generally experienced as easier to write, while the former provides more advanced functionalities. Existing pages can be adjusted directly (editing them online or on your computer). When you want to create a new page, make sure to add the page to the `index.rst` in order to make the page part of the website.
```

Comment (Beramos) [reply docs/collaborate.md:46]: fix typos

**Comment by Beramos** [comment]
This procedure is clear and works,  for me personally I still have a problem with the `pip install -e .[develop]` command but when installing the develop dependencies manually it works.

**Comment by Beramos** [comment]
closes #69

**Comment by jorisvandenbossche** [comment]
>  I still have a problem with the pip install -e .[develop] command

What error does it give?

**Comment by Beramos** [comment]
```
zsh: no matches found:  .[develop]. 
```

**Comment by stijnvanhoey** [comment]
```
pip install ".[develop]"
```

**Comment by jorisvandenbossche** [comment]
That seems a zsh issue, see eg https://stackoverflow.com/questions/30539798/zsh-no-matches-found-requestssecurity

**Comment by Beramos** [comment]
The quotation marks did the trick!

**Pull request merged**